vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -86,94 +88,105 @@ Use `ent help` to review the list of available commands.
 ``` sh
 rm -rf ~/.entando.
 ```
 
 ## Bundle Commands
-Use the `ent bundler` command to prepare a bundle for publication or extract a bundle from an application.
-1. Prepare a bundle custom resource from a Git repository. The project command (`ent prj generate-cr`) provides a wrapped version of this command. See the help for options including the bundle name, description, repository, etc. The output of this command is a yaml file which can be piped to a file or directly to `ent kubectl` for application to Kubernetes.
+Use the `ent bundler` command to prepare a bundle for publication or extract a bundle from an application
+
+### Prepare a Bundle for Publication
+
+The project command `ent prj generate-cr` provides a wrapped version of `ent bundler` and prepares a bundle custom resource from a Git repository. Reference the help text for options (e.g. bundle name, description, repository). The output of `ent prj generate-cr` is a YAML file which can be piped to `ent kubectl` for direct application to Kubernetes.
+
 ``` sh
   ent bundler from-git
 ```   
-See [this tutorial](../../tutorials/ecr/publish-simple-bundle.md) for an example using this command.
+See [this tutorial](../../tutorials/ecr/publish-simple-bundle.md) for an example using `ent prj generate-cr`.
+
+### Extract a Bundle from an Application
+
+Point the bundler to an existing Entando Application to extract its components (pages, content, etc.) and static assets into a custom bundle. This bundle can be used to migrate Entando components from one environment to another (e.g. Dev to QA), as a template for building a new Entando Application, or as the skeleton of an Entando solution. 
 
-2. Point the bundler to an existing Entando application and extract its components (pages, content, etc.) and static assets into a custom bundle. You can use this bundle to migrate Entando components from one environment to another (e.g. Dev to QA), to provide a template for building a new Entando application, or as the skeleton of an Entando solution. The bundler provides an interactive mode which allows you to identify the components to be exported from the application. The output of this command is the same bundle folder structure created by an Entando project including a top-level descriptor file.
+The bundler provides an interactive mode to identify the components to be exported from the application. The bundle folder structure created by an Entando project, including a top-level descriptor file, is output by the following command.
 ``` sh
   ent bundler from-env  
 ```
 
-You will need to provide an `env.json` file in the same directory where the bundler is run. This is used to configure the application URLs and client credentials.
+An `env.json` file to configure the application URLs and client credentials must live in the directory from which the bundler is run.
```

Comment:
No issues here. I keep forgetting to bring it up in our meetings, but Nathan was completely right about the article a/an, it follows sound only.

Reply:
awesome! thanks for following up on that :)